src/mlpack/methods/lmnn/lmnn_main.cpp

```diff
@@ -136,6 +138,8 @@ PARAM_INT_IN("rank", "Rank of distance matrix to be optimized. ", "A", 0);
 PARAM_FLAG("normalize", "Use a normalized starting point for optimization. It"
     "is useful for when points are far apart, or when SGD is returning NaN.",
     "N");
+PARAM_FLAG("center", "Perform mean-centering on the dataset. It is useful "
+    "when points have large norm values.", "C");
```

Comment:
I guess technically it would be `when the centroid of the data is far from the origin`.

Reply:
Right, that's should be the one. Thanks for pointing this out.

Reply:
Right, that should be the one. Thanks for pointing this out.